docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -93,14 +93,14 @@ This action on a project (like an ASP.NET Web application or Web API service) ad
 
 You usually build a custom image for your container on top of a base image you can get from an official repository at the [Docker Hub](https://hub.docker.com/) registry. That is precisely what happens under the covers when you enable Docker support in Visual Studio. Your Dockerfile will use an existing aspnetcore image.
 
-Earlier we explained which Docker images and repos you can use, depending on the framework and OS you have chosen. For instance, if you want to use ASP.NET Core and Linux, the image to use is microsoft/aspnetcore:1.1. Therefore, you just need to specify what base Docker image you will use for your container. You do that by adding FROM microsoft/aspnetcore:1.1 to your Dockerfile. This will be automatically performed by Visual Studio, but if you were to update the version, you update this value.
+Earlier we explained which Docker images and repos you can use, depending on the framework and OS you have chosen. For instance, if you want to use ASP.NET Core (Linux or Windows), the image to use is microsoft/aspnetcore:2.0. Therefore, you just need to specify what base Docker image you will use for your container. You do that by adding FROM microsoft/aspnetcore:2.0 to your Dockerfile. This will be automatically performed by Visual Studio, but if you were to update the version, you update this value.
```

Comment:
Usually it is better to have control over the specific version you are working with. For instance, when creating a project in VS 2017 with a .NET Core 2.0 project, when enabling Docker it will use the :2.0 tag, not the :latest. Specific versions are more predictable. :latest could. eventually, break your app or not.